test_scripts/Defects/5_0/1006/017_PerformInteraction_BOTH_mode_UI_isReady_false.lua

```diff
@@ -0,0 +1,105 @@
+-- - respond PerformInteraction with (resultCode: TIMED_OUT, success: false) to App in case user does not provide choice
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require("test_scripts/Defects/5_0/1006/common")
+
+--[[ Local Functions ]]
+local function sendPerformInteractionBothVrChoice(pParams)
+  pParams.interactionMode = "BOTH"
+  local cid = common.getMobileSession():SendRPC("PerformInteraction",pParams)
+  common.getHMIConnection():ExpectRequest("VR.PerformInteraction", {
+      helpPrompt = pParams.helpPrompt,
+      initialPrompt = pParams.initialPrompt,
+      timeout = pParams.timeout,
+      timeoutPrompt = pParams.timeoutPrompt
+    })
+  :Do(function(_,data)
+      common.getHMIConnection():SendNotification("TTS.Started")
+      common.getHMIConnection():SendNotification("VR.Started")
+      local function firstSpeakTimeOut()
+        common.getHMIConnection():SendNotification("TTS.Stopped")
+      end
+      common.runAfter(firstSpeakTimeOut, 1000)
+      local function vrResponse()
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+          { choiceID = pParams.interactionChoiceSetIDList[1] })
+        common.getHMIConnection():SendNotification("VR.Stopped")
+      end
+      common.runAfter(vrResponse, 2000)
+    end)
+  common.getHMIConnection():ExpectRequest("UI.PerformInteraction")
+  :Times(0)
+  common.getMobileSession():ExpectResponse(cid, {
+    success = true,
+    resultCode = "SUCCESS",
+    choiceID = pParams.interactionChoiceSetIDList[1],
+    triggerSource = "VR",
+    info = "UI is not supported by system"
+  })
+end
+
+local function sendPerformInteractionBoth(pParams)
+  pParams.interactionMode = "BOTH"
+  local cid = common.getMobileSession():SendRPC("PerformInteraction", pParams)
+  common.getHMIConnection():ExpectRequest("VR.PerformInteraction", {
+      helpPrompt = pParams.helpPrompt,
+      initialPrompt = pParams.initialPrompt,
+      timeout = pParams.timeout,
+      timeoutPrompt = pParams.timeoutPrompt
+    })
+  :Do(function(_, data)
+      common.getHMIConnection():SendNotification("VR.Started")
+      common.getHMIConnection():SendNotification("TTS.Started")
+      local function firstSpeakTimeOut()
+        common.getHMIConnection():SendNotification("TTS.Stopped")
+      end
+      common.runAfter(firstSpeakTimeOut, 5)
+      local function vrResponse()
+        common.getHMIConnection():SendError(data.id, data.method, "TIMED_OUT", "Perform Interaction error response")
+        common.getHMIConnection():SendNotification("VR.Stopped")
+      end
+      common.runAfter(vrResponse, 20)
+    end)
+  common.getHMIConnection():ExpectRequest("UI.PerformInteraction")
+  :Times(0)
+
+  common.getMobileSession():ExpectResponse(cid, {
+    success = false,
+    resultCode = "TIMED_OUT",
+    info = "Perform Interaction error response"
```

Comment:
I think that SDL should concatenate "UI is not supported by system" message to error message from VR response